### PR TITLE
[Docs] Fix example of how to set rev share precentage

### PR DESCRIPTION
**Pull request by Olshansk**
The existing docs for setting `rev_share_percentage` resulted in this error: `"cannot unmarshal !!seq into map[string]float32"`

### CODE REVIEW

File: localnet/poktrolld/config/supplier1_stake_config.yaml [hunk 1]
```diff
@@ -2,17 +2,17 @@ owner_address: pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4
...
+  pokt1eeeksh2tvkh7wzmfrljnhw4wrhs55lcuvmekkw: 19.5
 services:
   - service_id: anvil
     endpoints:
       - publicly_exposed_url: http://relayminer1:8545
         rpc_type: JSON_RPC
   - service_id: ollama
     rev_share_percent:
-      - pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4: 50
-      - pokt1eeeksh2tvkh7wzmfrljnhw4wrhs55lcuvmekkw: 50
```
Comment (red-0ne) [inline localnet/poktrolld/config/supplier1_stake_config.yaml:15]: Thanks for the fix. I wonder how this did work in the past :thinking:

**Review by red-0ne — APPROVED** [review]

File: localnet/poktrolld/config/supplier1_stake_config.yaml (hunk 1 above)
Comment (red-0ne) [reply localnet/poktrolld/config/supplier1_stake_config.yaml:15]: It did not work :sweat_smile:

Comment (Olshansk) [reply localnet/poktrolld/config/supplier1_stake_config.yaml:15]: How did you test it?

Comment (red-0ne) [reply localnet/poktrolld/config/supplier1_stake_config.yaml:15]: These specific config files are not being used. E2E tests are generating their own.

**Pull request merged**